ddp-workspace/projects/ddp-dsm-ui/src/app/utils/utils.ts

```diff
@@ -15,6 +15,7 @@ import { Filter } from '../filter-column/filter-column.model';
 import { AbstractionField } from '../medical-record-abstraction/medical-record-abstraction-field.model';
 import { Participant } from '../participant-list/participant-list.model';
 import { NameValue } from './name-value.model';
+import {AnswerValue} from "../../../../ddp-sdk/src/lib/models/activity/answerValue";
```

Comment:
`"` need to get changed to `'` please

Reply:
@LKapanad, please run `ng lint ddp-dsm-ui`  from` ./ddp-workspcase `in order to see if any linter error occurs. You might have some. Thanks.